wallet/src/main/java/com/android/identity_credential/wallet/ui/QrEngagementScreen.kt

```diff
@@ -0,0 +1,20 @@
+package com.android.identity_credential.wallet.ui
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.asImageBitmap
+import com.android.identity_credential.wallet.encodeQRCodeAsBitmap
+
+@Composable
+fun QrEngagementScreen(deviceEngagementUriEncoded: String) {
+     Box(modifier = Modifier.fillMaxSize()) {
+         Image(
+             bitmap = encodeQRCodeAsBitmap(deviceEngagementUriEncoded).asImageBitmap(),
+             contentDescription = "QR Code",
+             modifier = Modifier.align(Alignment.Center))
+     }
+}
```

Comment:
Why isn't this using `ScreenWithAppBarAndBackButton()`? I don't think it's very user-friendly to just show a QR code without any title bar or back button...